.circleci/config.yml

```diff
@@ -0,0 +1,244 @@
+  # circleci CLI tool.
+  # https://discuss.circleci.com/t/bug-circleci-build-command-ignores-checkout-path-config/13004
+  working_directory: /var/www/html/modules/apigee_m10n
+
+# YAML does not support merging of lists. That means we can't have a default
+# 'steps' configuration, though we can have defaults for individual step
+# properties.
+
+# We use the composer.json as a way to determine if we can cache our build.
+restore_cache: &restore_cache
+  keys:
+  - v4-dependencies-{{ checksum "composer.json" }}-{{ checksum "../../composer.json" }}
+  # fallback to using the latest cache if no exact match is found
+  - v4-dependencies-
+
+# If composer.json hasn't changed, restore the Composer cache directory. We
+# don't restore the lock file so we ensure we get updated dependencies.
+save_cache: &save_cache
+  paths:
+    - /root/.composer/cache/files
+  key: v4-dependencies-{{ checksum "composer.json" }}-{{ checksum "../../composer.json" }}
+
+# Install composer dependencies into the workspace to share with all jobs.
+update_dependencies: &update_dependencies
+  <<: *defaults
+  steps:
+    - checkout
+
+    - restore_cache: *restore_cache
+
+    - run:
+        working_directory: /var/www/html
+        command: |
+          cp ./modules/apigee_m10n/.circleci/RoboFile.php /var/www/html
+          cp ./modules/apigee_m10n/.circleci/update-dependencies.sh /var/www/html
+          ./update-dependencies.sh apigee_m10n
+
+    - save_cache: *save_cache
+
+    - persist_to_workspace:
+        root: /var/www/html
+        paths:
+          - .
+
+# Run Drupal unit and kernel tests as one job. This command invokes the test.sh
+# hook.
+unit_kernel_tests: &unit_kernel_tests
+  <<: *defaults
+  steps:
+    - attach_workspace:
+        at: /var/www/html
+
+    - checkout
+
+    - run:
+        working_directory: /var/www/html
+        command: |
+          cp ./modules/apigee_m10n/.circleci/update-dependencies.sh /var/www/html
+          cp ./modules/apigee_m10n/.circleci/RoboFile.php /var/www/html
+          ./test.sh apigee_m10n
+
+    - store_test_results:
+        path: /var/www/html/artifacts/phpunit
+    - store_artifacts:
+        path: /var/www/html/artifacts
+
+# Run Drupal functional tests. This command invokes the test-functional.sh
+# hook.
+functional_tests: &functional_tests
+  <<: *defaults
+  steps:
+    - attach_workspace:
+        at: /var/www/html
+
+    - checkout
+
+    - run:
+        working_directory: /var/www/html
+        command: |
+          cp ./modules/apigee_m10n/.circleci/update-dependencies.sh /var/www/html
+          cp ./modules/apigee_m10n/.circleci/RoboFile.php /var/www/html
+          ./test-functional.sh apigee_m10n
+
+    - store_test_results:
+        path: /var/www/html/artifacts/phpunit
+    - store_artifacts:
+        path: /var/www/html/artifacts
+
+# Run Drupal functional tests. This command invokes the test-functional-js.sh
+# hook.
+functional_js_tests: &functional_js_tests
```

Comment:
Note: This is going to test with chrome using selenium server. See: https://github.com/deviantintegral/drupal_tests/blob/master/hooks/test-functional-js.sh